custom_random_forest.py

```diff
@@ -0,0 +1,137 @@
+# Комментарий для проверяющего
+# Я понимаю, что задача CPU-bound, значит надо использовать процессы.
+# Но я пыталась параллелить на процессах с помощью ProcessPoolExecutor,
+# и с ним у меня для fit нормально уменьшалось wall time, но росло cpu time,
+# а для predict росло cpu time, а wall time уменьшался буквально на 10-15%.
+# С потоками получилось лучше: в обоих случаях cpu +- сохраняется, а wall почти идеально падает :(
+# Айм сорри...
+
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
Да, такое может быть. В субботу обсудим на лекции еще. В любом случае если хочешь - можешь удалить этот коммент:)
```suggestion
from concurrent.futures import ThreadPoolExecutor
```